TTBot/Models/ExcelSheetMappingModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TTBot.Models
+{
+    class ExcelSheetMappingModel
+    {
+        public static Dictionary<string, string> championshipsMappings = new Dictionary<string, string>()
+        {
+            {"ACC ProAmChampionship", "ACC GT3 8pm"},
```

Comment:
I'd like to move this config to the DB at some point if possible, so we can avoid code changes when theres new championships.

Reply:
Yeah 100% agree. At some point I'll create a command for adding new aliases and mappings to the DB from the bot